pkg/api/customization/namespace/namespace.go

```diff
@@ -66,13 +68,17 @@ func (w ActionWrapper) ActionHandler(actionName string, action *types.Action, ap
 	case "move":
 		clusterID, _ := ref.Parse(convert.ToString(actionInput["projectId"]))
 		userContext, err := w.ClusterManager.UserContext(clusterID)
-		if err != nil {
+		if err != nil && !kerrors.IsNotFound(err) {
```

Comment:
I think it could be written like this:
```
if err != nil {
			if !kerrors.IsNotFound(err) {
				return err
			}
			return httperror.NewAPIError(httperror.NotFound, err.Error())
		}